create doc for baseline contributions (#1003)
adap/flower: doc/source/index.rst

   changelog
   faq

> Baselines
> ---------

Reviewer: ```suggestion
Flower Baselines
----------------
```

> Baselines are a set of organized scripts used to reproduce results from well-known publications or benchmarks. You can check which baselines already exist and/or contribute your own baseline.

Reviewer: ```suggestion
Flower Baselines are a collection of organised scripts used to reproduce results from well-known publications or benchmarks. You can check which baselines already exist and/or contribute your own baseline.
```

.. toctree::
   :maxdepth: 1
   :caption: Baselines
   
   execute-baselines

>    contribute-baselines

Reviewer: ```suggestion
   contributing-baselines
```
